packages/ui/src/components/Badge/index.tsx

```diff
@@ -158,28 +158,53 @@ export const Badge = ({
   const sizeValue = SIZES[size]
 
   return (
-    <StyledBox
-      role="status"
-      aria-label={ariaLabel}
-      as="span"
-      sentiment={
-        disabled ? generatedStyles['disabled'] : generatedStyles[sentiment]
-      }
-      size={sizeValue}
-      className={className}
-      data-testid={dataTestId}
-    >
-      {icon && sizeValue !== SIZES.small ? (
-        <Icon name={icon} size={16} />
-      ) : null}
-      <StyledText
-        variant="bodyStrong"
-        as="p"
-        fontSize={TEXT_SIZES[size]}
-        prominence={disabled ? 'weak' : 'default'}
+    <>
+      <StyledBox
+        role="status"
+        aria-label={ariaLabel}
+        as="span"
+        sentiment={
+          disabled ? generatedStyles['disabled'] : generatedStyles[sentiment]
+        }
+        size={sizeValue}
+        className={className}
+        data-testid={dataTestId}
       >
-        {children}
-      </StyledText>
-    </StyledBox>
+        {icon && sizeValue !== SIZES.small ? (
+          <Icon name={icon} size={16} />
+        ) : null}
+        <StyledText
+          variant="bodyStrong"
+          as="p"
+          fontSize={TEXT_SIZES[size]}
+          prominence={disabled ? 'weak' : 'default'}
+        >
+          {children}
+        </StyledText>
+      </StyledBox>
+      <StyledBox
```

Comment:
I'm not sure to understand why it's being duplicated?

Reply:
i forgot to remove this test 🙈